superset/viz.py

```diff
@@ -1159,19 +1159,11 @@ def get_data(self, df):
                 l = [str(s) for s in name[1:]]
                 series_title = ", ".join(l)
             values = []
-            for i, v in ys.iteritems():
-                idx = pt.index[i]
-                if isinstance(idx, (tuple, list)):
-                    idx = ', '.join([str(s) for s in idx])
-                else:
-                    idx = str(idx)
-                values.append({
-                    'x': idx,
-                    'y': v,
-                })
             d = {
                 "key": series_title,
-                "values": values,
+                "values": [
```

Comment:
Not sure if I'm missing something. I think it was broken before because we used dataframe instead of pivot table when iterating, and pt.index is self.groupby, which doesn't include the breakdown columns. If we iterate on pivot table, index will give us the right labels